### PR TITLE
Component: chirp-box

### PR DESCRIPTION
**Pull request by JME469**
# Chirp Container component
## How to use:
component-use

1. Link the css dependencies from the image above.

2. Use the HTML custom tag: `<chirp-box></chirp-box>`

example

example2

### CODE REVIEW

File: public/css/chirpbox.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+3&family=Syne:wght@400;600&display=swap");
```
Comment (KaruzG) [inline public/css/chirpbox.css:1]: This font will be used in every page, I added it to index.css

I approve but this will need to be removed later

**Review by KaruzG — APPROVED** [review]
Looks good 👍

Needs fix on the dropdown, maybe use position absolute+zindex so the text below doesnt move.

Try not to upload test pages like feed.html

**Comment by Rafval18** [comment]
Good Job

**Review by Rafval18 — APPROVED** [review]
Ok.  That seems fine to me.

**Comment by KaruzG** [comment]
Fixed the conflicts, now I merge

**Pull request merged**